homeassistant/components/maico/connection.py

```diff
@@ -0,0 +1,31 @@
+"""Connection manaement."""
+import logging
+
+import httpx
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Connection:
+    """Class to manage Connection."""
```

Comment:
This doesn't tell the reader much anything. A class called `Connection` would presumably manage a connection... yet this doesn't really seem to do that. Instead it just encapsulates a httpx client and a host (string)?